apps/freelance/src/pages/ProfileQuestions2/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { useTranslation } from 'react-i18next';
+import { Wrapper } from './styles';
+import { Link } from 'react-router-dom';
+import { ProgressBar } from '@freelance/components';
+
+const ProfileQuestions2 = () => {
+  const { t } = useTranslation();
+  const progressBarPer = 50;
+  return (
+    <Wrapper>
+      <div>{t('description.profile p2 completion per')}</div>
+      <ProgressBar percent={progressBarPer} strokeColor={'#021691'} trailColor={'#B0C4DE'}/>
```

Comment:
all colors should be in constant file PLEASE check documents that I have been attached

Reply:
fixed. Created constant folder and values are taken from there. Screenshot can be seen above (progress bar colors)

Reply:
![image](https://user-images.githubusercontent.com/95277626/200795398-454a6034-91e9-434b-8347-7759b0d7f2fe.png)